### ISSUE TITLE
Crash with Enderscape

### ISSUE THREAD

**Issue by emoace** [LambdAurora/AurorasDecorations#14]
This might be due to the new wood type added in Enderscape not registering correctly with AurorasDecorations' log piles and other wood integration.  Not sure if it is on Enderscape's side or AurorasDecorations side though.

Attached are 2 crash reports, it mentions Iris in the second one, but I have used Iris/Sodium+Shaders with both Enderscapeand AurorasDecorations, separately, just fine with seemingly no errors in the log.

[crash-2021-10-28_13.58.14-server.txt](https://github.com/LambdAurora/AurorasDecorations/files/7436766/crash-2021-10-28_13.58.14-server.txt)
[crash-2021-10-28_13.58.15-client.txt](https://github.com/LambdAurora/AurorasDecorations/files/7436767/crash-2021-10-28_13.58.15-client.txt)

**Comment by emoace** [comment]
A similar issue presents itself when I add Adorn to the mix, which leads me to assume the underlying problem is on Enderscape's side.  I attached another crash report if you want to check it too.  I also removed NotEnoughCrashes from the mix because it seemed like it was just complicating things.

[crash-2021-10-28_14.08.53-client.txt](https://github.com/LambdAurora/AurorasDecorations/files/7436788/crash-2021-10-28_14.08.53-client.txt)
